sklearn/tree/_splitter.pyx

```diff
@@ -20,6 +20,14 @@ import numpy as np
 
 from scipy.sparse import csc_matrix
 
+from ..utils._sorting cimport (
+    sort,
+    swap,
+    median3,
+    introsort,
+    heapsort
+)
```

Comment:
I have the impression that only the `sort` function is called in the `_splitter.pyx` file:

```suggestion
from ..utils._sorting cimport sort
```